erpnext/accounts/report/financial_statements.html

```diff
@@ -13,7 +13,16 @@
 	}
 
 	.financial-statements-blank-row td {
-		height: 37px;
+		height: 20px;
```

Comment:
This is more of a duplication of logic in JS. can we print directly without using this template.